rll-examples/install-packages.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# ---
+# RightScript Name: SYS packages install
+# Description: |
+#   Installs packages required by RightScripts.
+#   To handle naming variations, prefix package names with "yum:" or "apt:".
+# Inputs:
+#   PACKAGES_INSTALL:
```

Comment:
Hmm. Since the primary "input" is `RS_PACKAGES`, I was hoping to choose similar names for the actual inputs. Of course, inputs can't begin with `RS_`.

I also wanted to follow the reverse-lexical naming scheme that we _used_ to have for scripts and inputs: `COLLECTD_ENABLE`, `RUBY_GEM_VERSION`, that sort of thing ... this kind of "natural sort" can be super useful if someone is looking at an alphabetized list of inputs: the ones that have something to do with each other, are next to each other.

However, looking at the `rll` folder, it seems like we've moved beyond that convention to plain-English input names: `MONITORING_METHOD`, `COLLECTD_SERVER`, `RIGHTLINK_DOCKER`, and so forth.

I guess we could switch this to `INSTALL_PACKAGES` ... I would omit the `ENABLE` word because I don't see any other scripts that use `ENABLE` in their names to indicate a level of enablement. The closest we have is `RIGHTLINK_DOCKER`.

Another option is to remove this input entirely _a la_ managed login: if you don't want to install packages, then don't add the script to your boot bundle!

Your choice?